plugins/modules/rhevm.py

```diff
@@ -8,151 +8,150 @@
-        type: list
-        elements: str
-        default: [ hd, network ]
-    del_prot:
-        description:
-            - This option sets the delete protection checkbox.
-        type: bool
-        default: true
-    cd_drive:
-        description:
-            - The CD you wish to have mounted on the VM when O(state=cd).
-        type: str
-    timeout:
-        description:
-            - The timeout you wish to define for power actions.
-            - When O(state=up).
-            - When O(state=down).
-            - When O(state=restarted).
-        type: int
-'''
-
-RETURN = r'''
+  user:
+    description:
+      - The user to authenticate with.
+    type: str
+    default: admin@internal
+  password:
+    description:
+      - The password for user authentication.
+    type: str
+    required: true
+  server:
+    description:
+      - The name/IP of your RHEV-m/oVirt instance.
+    type: str
+    default: 127.0.0.1
+  port:
+    description:
+      - The port on which the API is reachable.
+    type: int
+    default: 443
+  insecure_api:
+    description:
+      - A boolean switch to make a secure or insecure connection to the server.
+    type: bool
+    default: false
+  name:
+    description:
+      - The name of the VM.
+    type: str
+  cluster:
+    description:
+      - The RHEV/oVirt cluster in which you want you VM to start.
+    type: str
+    default: ''
+  datacenter:
+    description:
+      - The RHEV/oVirt datacenter in which you want you VM to start.
+    type: str
+    default: Default
+  state:
+    description:
+      - This serves to create/remove/update or powermanage your VM.
+    type: str
+    choices: [absent, cd, down, info, ping, present, restarted, up]
+    default: present
+  image:
+    description:
+      - The template to use for the VM.
+    type: str
+  type:
+    description:
+      - To define if the VM is a server or desktop.
+    type: str
+    choices: [desktop, host, server]
+    default: server
+  vmhost:
+    description:
+      - The host you wish your VM to run on.
+    type: str
+  vmcpu:
+    description:
+      - The number of CPUs you want in your VM.
+    type: int
+    default: 2
+  cpu_share:
+    description:
+      - This parameter is used to configure the CPU share.
+    type: int
+    default: 0
+  vmmem:
+    description:
+      - The amount of memory you want your VM to use (in GB).
+    type: int
+    default: 1
+  osver:
+    description:
+      - The operating system option in RHEV/oVirt.
+    type: str
+    default: rhel_6x64
+  mempol:
+    description:
+      - The minimum amount of memory you wish to reserve for this system.
+    type: int
+    default: 1
+  vm_ha:
+    description:
+      - To make your VM High Available.
+    type: bool
+    default: true
+  disks:
+    description:
+      - This option uses complex arguments and is a list of disks with the options O(name), O(size), and O(domain).
```

Comment:
I reckon these "options" mentioned are actually non-declared suboptions. Changing the markup to `V()`.
```suggestion
      - This option uses complex arguments and is a list of disks with the options V(name), V(size), and V(domain).
```